portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -72,10 +72,17 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 </template>
 <script>
 export default {
+  props: {
+    earnerId: {
+      type: Number,
+      default: -1,
+    }
```

Comment:
1. make the default value of `earnerId` prop as a function (Vue best practices)
2. Add a boolean prop to know whether we will retrieve all users realizations or just for current user (we may need to have this boolean configured by admin using switch button in the future by example. This will allow more flexibility)

```suggestion
    earnerId: {
      type: Number,
      default: () => 0,
    },
    retrieveAll: {
      type: Boolean,
      default: false,
    },
```